src/AssignTransfer/containers/OffenderResultsContainer.js

```diff
@@ -37,6 +37,27 @@ class OffenderResultsContainer extends Component {
     this.doSearch()
   }
 
```

Comment:
Lint was saying some functions needed moving above/below others, hence the changes here and elsewhere.